integration-tests/gradle/src/test/java/io/quarkus/gradle/LaunchUtils.java

```diff
@@ -21,4 +23,18 @@ protected static Process launch(Path jar, File output) throws IOException {
         return processBuilder.start();
     }
 
+    public static void dumpFileContentOnFailure(final Callable<Void> operation, final File logFile,
+                                                   final Class<? extends Throwable> failureType) throws Exception {
+
+        final Logger log = Logger.getLogger(FastJarFormatWorksTest.class);
```

Comment:
Sorry, I missed this. You shouldn't this. A log is always per class. So just add 

```
private static final Logger log = Logger.getLogger(LaunchUtils.class);
```

as a field of `LaunchUtils` and that in this method instead of creating the log in the method

Reply:
After making the change and ensuring that the test passes, you can mark the PR as ready for review

Reply:
```private static final Logger log``` gives two errors to make Logger not static and not private

Reply:
Here is an example of what I mean: https://github.com/quarkusio/quarkus/blob/259758b0d9667451b0d4522b88aa15f5b4a8020f/extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AmazonLambdaRecorder.java#L30

Reply:
Got it, I was doing it inside the method that's why had the error